### ISSUE TITLE
Getting NoSuchIdentifierException when using custom CRS

### ISSUE THREAD

**Issue by chrudosvorlicek** [mapfish/mapfish-print#188]
I've added new CRS (EPSG:5514 definition: http://epsg.io/5514.prettywkt ) to epsg.properties. When  Printing out map using this CRS results in: Error while processing request:
org.opengis.referencing.NoSuchIdentifierException: No transform for classification "Krovak (North Orientated)".

**Comment by LukasB79** [comment]
I have the same problem in MapFish 3.3.0 - any suggestions ?

**Comment by chrudosvorlicek** [comment]
Do you have problem with same coordinate system? If so, I solved it by using epsg 102067 (older code of 5514) in both data and configuration. for some reason mapfish cannot recognize 5514. If coordinate system is different try to find older version.

**Comment by LukasB79** [comment]
Yes, i have the same problem with the same coordinate system. I will try to use your advice. Thank you.

**Comment by chrudosvorlicek** [comment]
After some research and lot of code digging I found out, that problem itself lies in geotools and epsg definition. In epsg data file (from epsg.org) is the name of transformation "Krovak (North Orientated)" however the class with transformation code (in geotools) is called just "Krovak". When I changed name of transformation to "Krovak", I was able to print in EPSG:5514. 

As this is not problem of Mapfish print itself, I'm closing this issue.
